grammars/src/main/antlr/adl_rules.g4

```diff
@@ -63,7 +63,7 @@ booleanConstraintExpression
     | equalityExpression;
 
 
-booleanConstraint: adlRulesPath SYM_MATCHES ('{' c_primitive_object '}' | CONTAINED_REGEXP );
+booleanConstraint: equalityExpression SYM_MATCHES ('{' c_primitive_object '}' | CONTAINED_REGEXP );
```

Comment:
This should be very backwards compatible: the and/or/xor things above aren't affected, so no change in existing rules.